docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,192 @@
+- [Testing Your Changes](#testing-your-changes)
+- [Creating a Pull Request](#creating-a-pull-request)
+
+## Prerequisites
+
+In order to contribute to `ITKRemoteModuleBuildTestPackageAction` you
+must have the following:
+- A [GitHub](https://github.com/) account;
+- A PC with a [git](https://git-scm.com/) client and a text editor of your choice.
+
+The following are also helpful, but not strictly required:
+- A PC running the operating system for which your changes are targeted,
+  i.e. Windows, Ubuntu (Linux), or MacOS. This will allow you to test changes locally
+  before pushing to GitHub.
+- A Python interpreter of 3.7 or greater for testing wheels locally.
+
+Before getting started it's helpful to have a working understanding of the following:
+- ITK external modules
+  (see [README discussion](../README.md#motivation));
+- [GitHub Actions](https://docs.github.com/en/actions) and
+  [continuous integration](https://docs.github.com/en/actions/automating-builds-and-tests/about-continuous-integration)
+  in general;
+- [GitHub Actions Reusable Workflows](https://docs.github.com/en/actions/using-workflows/reusing-workflows).
+
+The following tools and concepts are used in `ITKRemoteModuleBuildTestPackageAction`
+and may be helpful or required for contributing:
+
+- [Bash](https://www.gnu.org/software/bash/) and
+  [PowerShell](https://learn.microsoft.com/en-us/powershell/) scripting;
+- [CMake](https://cmake.org/) and [Ninja](https://ninja-build.org/) build tools;
+- [CTest](https://cmake.org/cmake/help/latest/manual/ctest.1.html) and 
+  [CDash](https://cmake.org/cmake/help/book/mastering-cmake/chapter/CDash.html)
+  testing and reporting tools;
+- [Python](https://www.python.org/) scripting;
+- [Python wheel](https://wheel.readthedocs.io/en/stable/) packaging;
+
+## Creating or Modifying a Workflow
+
+GitHub Actions requires that workflows are placed in the [`.github/workflows`](../.github/workflows) directory.
+This is where you will find existing workflows and possibly add new workflows.
+
+To contribute to `ITKRemoteModuleBuildTestPackageAction` you will need to
+first [fork](https://docs.github.com/en/github-ae@latest/get-started/quickstart/fork-a-repo)
+the repository to your user account on GitHub and then
+[clone](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository)
+the fork to your local machine.
+
+### Creating a Workflow
+
+`ITKRemoteModuleBuildTestPackageAction` is a central repository for Github Actions
+reusable workflows for ITK external modules. Before creating a workflow, ask yourself
+the following questions:
+1. Does my workflow address a problem for ITK external modules that is not
+   already solved?
+2. Does my workflow provide functionality that will generalize to various ITK external modules?
+3. Does my workflow provide functionality related to continuous integration such as
+   building, packaging, and/or testing steps?
```

Comment:
We can use 1. for all numbers to avoid requiring to change them if ever a new point is raised. Markdown takes care of counting.